components/admin_console/schema_admin_settings.jsx

```diff
@@ -904,4 +903,32 @@ export default class SchemaAdminSettings extends React.Component {
             </div>
         );
     }
+
+    render = () => {
+        const schema = this.props.schema;
```

Comment:
nit: We're almost consistent in doing like ``const {schema} = this.props;``; easier to add/remove variable if needs to change in the future.